widget/index.js

```diff
@@ -165,7 +179,10 @@ var WidgetGenerator = yeoman.generators.Base.extend({
       this.template('nls/_strings.js', path.join(basePath, 'nls/strings.js'));
     }
     this.copy('images/icon.png', path.join(basePath, 'images/icon.png'));
-    this.template('_manifest.json', path.join(basePath, 'manifest.json'));
+    if(this.is2d)
```

Comment:
Seems like there are minimal changes between the 2d/3d manifest.json files, so we should use a single template w/ inline conditionals.